db-migrations/13_add_survey_mode_column.up.sql

```diff
@@ -0,0 +1 @@
+ALTER type survey.survey_mode ADD VALUE IF NOT EXISTS 'EQ_AND_SEFT';
```

Comment:
I would of liked to do an existence check first on the survey_mode type, however even though that can be done without issue, you can't Do an ADD type in a function. Its a known restriction, create and rename no problem ADD you get a ```ERROR:  ALTER TYPE ... ADD cannot be executed from a function```